lib/ReactViews/Search/SearchBoxAndResults.jsx

```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import { removeMarker } from "../../Models/LocationMarkerUtils";
+import { reaction, runInAction } from "mobx";
+import { Trans, withTranslation } from "react-i18next";
+import PropTypes from "prop-types";
+import { observer } from "mobx-react";
+// import { ThemeContext } from "styled-components";
+
+import SearchBox from "../Search/SearchBox";
+// import SidebarSearch from "../Search/SidebarSearch";
+import LocationSearchResults from "../Search/LocationSearchResults";
+import Icon, { StyledIcon } from "../Icon";
+
+import Box from "../../Styled/Box";
+import Text from "../../Styled/Text";
+import Spacing from "../../Styled/Spacing";
+import { RawButton } from "../../Styled/Button";
+
+import { addMarker } from "../../Models/LocationMarkerUtils";
+
+export function SearchInDataCatalog({ viewState, handleClick }) {
+  const locationSearchText = viewState.searchState.locationSearchText;
+  return (
+    <RawButton
+      fullWidth
+      onClick={() => {
+        const { searchState } = viewState;
+        runInAction(() => {
+          // Set text here so that it doesn't get batched up and the catalog
+          // search text has a chance to set isWaitingToStartCatalogSearch
+          searchState.catalogSearchText = searchState.locationSearchText;
+        });
+        viewState.searchInCatalog(searchState.locationSearchText);
+        handleClick && handleClick();
+      }}
+    >
+      <Box paddedRatio={2} rounded charcoalGreyBg>
+        <StyledIcon styledWidth={"14px"} glyph={Icon.GLYPHS["dataCatalog"]} />
+        <Spacing right={2} />
+        <Text textAlignLeft textLight large fullWidth>
+          <Trans
+            i18nKey="search.searchInDataCatalog"
+            locationSearchText={locationSearchText}
+          >
+            Search <strong>{{ locationSearchText }}</strong> in the Data
+            Catalogue
+          </Trans>
+        </Text>
+        <StyledIcon glyph={Icon.GLYPHS.right2} styledWidth={"14px"} light />
```

Comment:
I was thinking if we needed to do something about this hardcoded width, since 14px seems to be be used repeatedly. Maybe we should make a bunch of global variables for common icon widths? 🤔 but that's just my very rough thoughts

Reply:
Yeah it's a side-effect of
- our current icon system
- the designs needing various sizes of icons